src/doctypes/todos.js

```diff
@@ -0,0 +1,5 @@
+export const TODOS_DOCTYPE = 'io.mocks.todos'
```

Comment:
Is it normal to have a TODOs doctype?

Reply:
I removed it, with other useless files in https://github.com/cozy/cozy-passwords/pull/1/commits/8d4edff684c63f0bdb71c48cb604f3acd42c2c29

Reply:
In that case, the test https://github.com/cozy/cozy-passwords/pull/1/files#diff-8bcc848f0322ea15effe1e2f25e25a3c should also be removed, no?

Reply:
Yep, thanks https://github.com/cozy/cozy-passwords/pull/1/commits/4bbe40f21b70b120f5ba87abbcf31eb4ae28f58b